apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -82,27 +111,38 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
     """
   end
 
-  def mount(
-        _params,
-        %{"is_curator" => is_curator, "ingestion" => ingestion, "user_id" => user_id} = _session,
+  def handle_info(
+        {:updated_metadata,
+          %Ecto.Changeset{ data: %AndiWeb.InputSchemas.IngestionMetadataFormSchema{} } = metadata_changeset },
```

Comment:
A nice "trick" for specifying the type of changeset passed by uses the data atom to match the type.

Especially nice when paired with the unhandled/catch_all handle_info/handle_event functions below.